enterprise/validators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Database models field validators.
+"""
+import os
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+def validate_image_extension(value):
```

Comment:
@asadiqbal08 please add unittests for this method (at least three: `png`, any other extension, no extension).